addons/calendar/models/res_users.py

```diff
@@ -81,11 +81,12 @@ def _compute_calendar_default_privacy(self):
         When any user doesn't have its setting from ResUsersSettings defined, fallback to Default User Template's.
         """
         fallback_default_privacy = 'public'
-        if any(not user.res_users_settings_id.calendar_default_privacy for user in self):
+        # sudo: any user has access to any other users settings
```

Comment:
This is still not correct to me, users shouldn't have access to other users settings. Just this setting in particular.